pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -186,6 +177,7 @@ func (a Adapter) Test(testCmd string, show bool) (err error) {
 
 	testCommand, err := common.ValidateAndGetTestDevfileCommands(a.Devfile.Data, testCmd)
 	if err != nil {
+		err.Error()
```

Comment:
also not sure the point of this over here

Reply:
oops. This was added by mistake, thought I was editing another branch. 
Removed.